internal/deviations/deviations.go

```diff
@@ -67,6 +67,8 @@ import "flag"
 
 // Vendor deviation flags.
 var (
+	MixedOcCliPrefersCli = flag.Bool("deviation_mixed_occli_prefers_cli", false,
```

Comment:
Why is this deviation added in this PR? The name also stutters. "MixedOriginPrefersCLI" would have been more accurate.

Reply:
Whoops, I botched that during a rebase. I'll make a PR to remove it.